src/api/transaction/transactionUtils.ts

```diff
@@ -84,6 +88,17 @@ export async function readArrowFiles(files: TransactionAsyncFile[]) {
   return results;
 }
 
+export async function readProtoMetadata(file: File | Blob) {
+  if (!file?.arrayBuffer) {
+    throw new Error(`Unsupported metadata type: ${typeof file}`);
+  }
```

Comment:
So this will happen on older engines, right? Should we have the error message tell users to upgrade their engines?

Reply:
No, it won't. It's looking for `metadata.proto` part, so it won't get here.